src/network/chart_cache_updater.py

```diff
@@ -304,6 +331,22 @@ def update_cache_scores():
             else:
                 live_data[key_str] = 0
         live_data['difficulty'] = live_data['diff']
+        total_notes = len(notes_data)
+        combo_thresholds = np.array([
+            1,
+            total_notes // 20,
+            total_notes // 10,
+            total_notes // 4,
+            total_notes // 2,
+            total_notes * 7 // 10,
+            total_notes * 8 // 10,
+            total_notes * 9 // 10,
+            total_notes + 1
+        ])
+        for timer in timers:
+            live_data['Timer_{}{}'.format(timer[0], timer[2])] = np.repeat(
+                [1., 1.1, 1.2, 1.3, 1.4, 1.5, 1.7, 2.], combo_thresholds[1:] - combo_thresholds[:-1]
```

Comment:
Try to not use magic numbers. Combo thresholds and bonuses are defined in `static.live_values`.